README.md

```diff
@@ -23,8 +23,8 @@ jobs:
           id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.PRIVATE_KEY }}
       - run: "echo installations: '${{ steps.stats.outputs.installations }}'"
-      - run: "echo suspended: '${{ steps.stats.outputs.suspendedInstallations }}'"
       - run: "echo most popular repositories: '${{ steps.stats.outputs.popular_repositories }}'"
+      - run: "echo stats: '${{ steps.stats.outputs.stats }}'"
```

Comment:
I don't think we need to add a `stats` output that just combines `installations` and `popular_repositories`. You can do that yourself as consumer of the action.

This is too Probot-specific. I created this action as a path to get stats for Probot users, but it's useful to everyone building GitHub Apps